iothub/service/src/Utilities/Logging.cs

```diff
@@ -4,13 +4,16 @@
 using System;
 using System.Collections;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.Diagnostics.Tracing;
 using System.Globalization;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
 namespace Microsoft.Azure.Devices
 {
+    /// Logging is not part of the core functionality of the client, so we will not concern ourselves with code coverage of this class.
+    [ExcludeFromCodeCoverage]
```

Comment:
I recall that we would like to not exclude this from code coverage analysis but lower its priority only, according to our discussion last time. What's your thought on this, @drwill-ms ?